ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx

```diff
@@ -58,6 +63,14 @@ const AppointmentEditorCommonFieldsWrapper = props => {
         }
     };
 
+    const getProviderErrorMessages = () => {
+        if (errors.providerError) {
+            return getMaxAppointmentProvidersErrorMessage(intl,
+                appConfig && appConfig.maxAppointmentProviders || DEFAULT_MAX_APPOINTMENT_PROVIDERS).providerErrorMessage;
+        }
+        if (errors.providerMandatoryError) return errorTranslations.providerMandatoryErrorMessage;
+    };
+
```

Comment:
@supriyatw is this a new error that we are handling as part of this piece of code ?

Reply:
yes @binduak there should be two errors 1. if the providers limit exceeds the max number of providers given in config  2. if the provider is mandatory and empty

Reply:
sure..this was missing and you have handled it in the code